packages/logger/package.json

```diff
@@ -0,0 +1,41 @@
+{
+    "name": "@sap-ux/logger",
+    "version": "0.1.0",
+    "description": "A simple logging module",
+    "license": "SEE LICENSE IN LICENSE",
+    "private": false,
+    "main": "dist/index.js",
+    "scripts": {
+        "build": "pnpm clean && tsc",
+        "watch": "tsc --watch",
+        "clean": "rimraf dist coverage",
+        "format": "prettier --write '**/*.{js,json,ts,yaml,yml}' --ignore-path ../../.prettierignore",
+        "lint": "eslint . --ext .ts",
+        "lint:fix": "eslint . --ext .ts --fix",
+        "test": "jest --maxWorkers=1 --colors",
+        "test-u": "jest --maxWorkers=1 --colors -u",
+        "link": "pnpm link --global",
+        "unlink": "pnpm unlink --global"
+    },
+    "dependencies": {
+        "chalk": "4.1.2",
+        "lodash.clonedeep": "4.5.0",
+        "logform": "2.3.2",
+        "winston": "3.3.3",
+        "winston-transport": "4.4.1"
+    },
+    "devDependencies": {
+        "@types/debug": "4.1.7",
+        "@types/lodash.clonedeep": "4.5.6",
```

Comment:
```suggestion
        "@types/lodash": "4.14.176"
```

Reply:
Fixed in https://github.com/SAP/open-ux-tools/pull/257/commits/34fab90d6a1a69d58b44c43d0be32b66836d41df